accounts/views.py

```diff
@@ -77,6 +77,17 @@ def crash_me(request):
     raise Exception
 
 
+def check_username(request):
+    username = request.GET.get('username', None)
+    res = {"result": False}
```

Comment:
Very, very minor comments on style: why create the dict here and not just say: `username_valid = False` and then at return write: `return JsonResponse(dict(result=username_valid))` ? This might be a lot more understandable when reading the code...